### PR TITLE
Ajouter des méta-données aux Aides (entité OpenFisca sous-jacent, type de la prestation)

### PR DESCRIPTION
**Pull request by guillett**
Depends on #584

### CODE REVIEW

File: app/js/constants/droits.js [hunk 1]
```diff
@@ -153,14 +169,18 @@ var droitsDescription = {
...
+                    'type': 'float',
                     'uncomputability': {
                         'tns': {
                             'reason': 'vous avez des revenus en tant qu’indépendant·e',
-                            'solution': 'Vous pouvez demander à bénéficier du RSA, mais c’est le président de votre conseil départemental qui <a target="_blank" rel="noopener" title="Article R262-23 du code de l’action sociale" href="https://www.legifrance.gouv.fr/affichCodeArticle.do?idArticle=LEGIARTI000028251799&cidTexte=LEGITEXT000006074069">décidera</a> de la manière dont vos revenus non salariés impacteront le montant de votre aide.'
+                            'solution': 'Vous pouvez demander à bénéficier du RSA, mais c’est le président de votre conseil départemental qui <a target="_blank" rel="noopener" title="Article R262-23 du code de l’action sociale" href="https://www.legifrance.gouv.fr/affichCodeArticle.do?idArticle=LEGIARTI000028251799&cidTexte=LEGITEXT000006074069">décidera</a> de la manière dont vos revenus non salariés impacteront le montant de votre aide.',
+                            'thirdPersonReason': 'le demandeur a des revenus en tant qu’indépendant·e',
```
Comment (MattiSG) [inline app/js/constants/droits.js]: Rather introduce a sub-object for `reason`.

File: app/js/constants/droits.js [hunk 2]
```diff
@@ -153,14 +169,18 @@ var droitsDescription = {
...
                         'conjoint_tns': {
                             'reason': 'votre conjoint·e a des revenus en tant qu’indépendant·e',
-                            'solution': 'Vous pouvez demander à bénéficier du RSA, mais c’est le président de votre conseil départemental qui <a target="_blank" rel="noopener" title="Article R262-23 du code de l’action sociale" href="https://www.legifrance.gouv.fr/affichCodeArticle.do?idArticle=LEGIARTI000028251799&cidTexte=LEGITEXT000006074069">décidera</a> de la manière dont les revenus non salariés de votre conjoint·e impacteront le montant de votre aide.'
+                            'solution': 'Vous pouvez demander à bénéficier du RSA, mais c’est le président de votre conseil départemental qui <a target="_blank" rel="noopener" title="Article R262-23 du code de l’action sociale" href="https://www.legifrance.gouv.fr/affichCodeArticle.do?idArticle=LEGIARTI000028251799&cidTexte=LEGITEXT000006074069">décidera</a> de la manière dont les revenus non salariés de votre conjoint·e impacteront le montant de votre aide.',
+                            'thirdPersonReason': 'le conjoint du demandeur a des revenus en tant qu’indépendant·e',
```
Comment (MattiSG) [inline app/js/constants/droits.js]: Where are those `thirdPersonReason` used?

Comment (guillett) [inline app/js/constants/droits.js]: Nowhere yet, I want to use them in the contribution view to generate test cases.

**Comment by guillett** [comment]
Superseeded by #628

**Pull request closed without merging**